main.js

```diff
@@ -44,20 +43,41 @@ function renderList(choiceList, choice) {
   }
 }
 
-function clearList(choiceList, choice) {
-  for (var i = 0; i < choice.length; i++) {
-    choiceList.removeChild(choice[i])
+function clearList(choiceList, choices) {
+  for (var i = 0; i < choices.length; i++) {
+    choiceList.removeChild(choices[i])
+  }
+}
+
+function randomizeList(choiceList, choices) {
+  var index = Math.round(Math.random() * (choices.length - 1))
+  console.log(index)
+
+  for (var i = 0; i < choices.length; i++) {
+    if (index !== i) {
+      choiceList.removeChild(choices[i])
```

Comment:
Is choiceList a DOM element? If so, I noticed you used $ in front of elements in other parts of the code. It is good to be consistent.